src/signals/incident-management/containers/IncidentOverviewPage/index.test.js

```diff
@@ -142,32 +176,40 @@ describe('signals/incident-management/containers/IncidentOverviewPage', () => {
       .find(IncidentOverviewPageContainerComponent)
       .props();
 
-    expect(containerProps.onRequestIncidents).not.toBeUndefined();
-    expect(typeof containerProps.onRequestIncidents).toEqual('function');
+    expect(containerProps.pageChangedAction).not.toBeUndefined();
+    expect(typeof containerProps.pageChangedAction).toEqual('function');
 
-    expect(containerProps.onPageIncidentsChanged).not.toBeUndefined();
-    expect(typeof containerProps.onPageIncidentsChanged).toEqual('function');
-
-    expect(containerProps.onChangeOrdering).not.toBeUndefined();
-    expect(typeof containerProps.onChangeOrdering).toEqual('function');
+    expect(containerProps.orderingChangedAction).not.toBeUndefined();
+    expect(typeof containerProps.orderingChangedAction).toEqual('function');
   });
 
-  it('should scroll page to top after navigating with pagination', () => {
+  it('should set page after navigating with pagination', () => {
     constants.FILTER_PAGE_SIZE = 30;
-    Object.defineProperty(window, 'scrollTo', {
-      value: () => {},
-      writable: true,
-    });
-    const scrollSpy = jest.spyOn(window, 'scrollTo');
 
-    const { getByTestId } = render(withAppContext(<IncidentOverviewPage />));
+    const incidents = generateIncidents();
 
-    fireEvent.click(
-      getByTestId('pagination').querySelector('button:first-of-type')
+    const { getByTestId } = render(
+      withAppContext(
+        <IncidentOverviewPageContainerComponent
+          {...props}
+          incidents={{
+            count: incidents.length,
+            results: incidents,
+            loading: false,
+          }}
+        />
+      )
     );
 
-    expect(scrollSpy).toHaveBeenCalledWith(0, 0);
-    expect(pageIncidentsChanged).toHaveBeenCalledWith(2);
+    expect(props.pageChangedAction).not.toHaveBeenCalled();
+
+    act(() => {
+      fireEvent.click(
+        getByTestId('pagination').querySelector('button:first-of-type')
+      );
+    });
+
+    expect(props.pageChangedAction).toHaveBeenCalledWith(expect.any(Number));
```

Comment:
Don't we know exactly what it's being called with here?

Reply:
We do, yes. Changing it.